src/app/hooks/useGlean.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useEffect } from "react";
+import Glean from "@mozilla/glean/web";
+import * as appEvents from "../../telemetry/generated/appEvents";
+
+// Custom hook that initializes Glean and returns the Glean objects required
+// to record data.
+export const useGlean = () => {
+  // Initialize Glean only on the first render
+  // of our custom hook.
+  useEffect(() => {
+    Glean.initialize("monitor-testing", true, {
+      // This will submit an events ping every time an event is recorded.
+      maxEvents: 1,
+    });
+
+    if (process.env.NODE_ENV === "development") {
+      // Enable logging pings to the browser console.
+      Glean.setLogPings(true);
+
+      // Set the tag and sends pings to the debug ping viewer.
+      Glean.setDebugViewTag("monitor-testing");
```

Comment:
I would recommend leaving this line out completely and enable things from the console as needed when manual QA is needed.

Reply:
I'm fine either way, we just want to make it easy for folks to see the data locally.